index.js

```diff
@@ -49,6 +53,9 @@ module.exports = {
   format: function(text, opts) {
     return formatWithShebang(text, normalizeOptions(opts));
   },
+  formatAST: function(ast, opts) {
```

Comment:
Hmm, this seems like a fairly core requirement for prettier to play nicely with other tooling, so I feel like it should be part of the core API - if it's in `__debug` no one will use it, right? Or maybe I've misunderstood?